src/api/app/models/workflow.rb

```diff
@@ -136,12 +136,7 @@ def destroy_target_projects
 
     target_project_names = processable_steps.map(&:target_project_name).uniq.compact
 
-    begin
-      Project.where(name: target_project_names).destroy_all
-    rescue Project::Errors::UnknownObjectError, Pundit::NotAuthorizedError => e
-      workflow_run.update_as_failed(e.message)
-      raise e
-    end
+    Project.where(name: target_project_names).destroy_all
```

Comment:
We are no longer covering the `Project::Errors::UnknownObjectError` now. Am I missing it somewhere?

Reply:
Yeah, it's an `APIError`.